TEST-SOURCEMAP-UPLOAD.md

```
@@ -0,0 +1,206 @@
+# You should see uploads to ALL projects in CHROME_SENTRY_PROJECTS:
+# - advisor-rhel
+# - compliance-rhel
+# - dashboard-rhel
+# - inventory-rhel
+# - etc.
+```
+
+### Expected Output
+
+Look for lines like:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)